codegen/util.go

```diff
@@ -132,17 +132,16 @@ func findField(typ *types.Struct, name, structTag string) (*types.Var, error) {
 		}
 
 		if field.Anonymous() {
-			if named, ok := field.Type().(*types.Struct); ok {
-				f, err := findField(named, name, structTag)
-				if err != nil && !strings.HasPrefix(err.Error(), "no field named") {
-					return nil, err
-				}
-				if f != nil && foundField == nil {
-					foundField = f
-				}
+
+			fieldType := field.Type()
+
+			if ptr, ok := fieldType.(*types.Pointer); ok {
+				fieldType = ptr.Elem()
```

Comment:
what happens when the embedded pointer is null? Worth adding a test?

Reply:
Yep, good question, nothing will happen here. But I would expect a runtime panic. What do you think we should do?

Reply:
can it return null and do the normal error bubbling dance?

Reply:
Yeah ok, I'll come up with a test.  Might involve slightly more messing around here since we'd need to know if it was a pointer that we matched to in template.

Reply:
Ok, after reviewing this a bit I _think_ this is going to be a bigger change.  Embeds can be pathological when they are more than a single embed deep with a mix of pointers thrown in.

I think for now we merge this and maybe log that as a separate issue.